src/GT.Core/Services/MappingProfiles/GTMappingProfile.cs

```diff
@@ -11,7 +11,7 @@ public GTMappingProfile()
 			CreateMap<Company, CompanyDTO>()
 				.ReverseMap();
 
-			CreateMap<Listing, ListingDTO>()
```

Comment:
Should have both ListingDTO and ListingPartialDTO.

Reply:
This will be managed in issue #67.